pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -187,9 +188,15 @@ def connection_save(context, name, **options):
     """
     Save the current connection to a new WBEM connection definition.
 
-    Create a new WBEM connection definition in the connections file from the
-    current connection. A connection definition with the name NAME must not yet
-    exist. The NAME argument (the name of the new connection) is required.
+    Save WBEM connection definition in the connections file from the current
+    connection. The NAME argument (the name of the new connection) is required.
```

Comment:
The first sentence of this paragraph could be streamlined, e.g. to "Save the current connection as a new connection definition in the connections file."

Reply:
DONE except for word new since it can overwrite an existing connection

DONE